kin-ecosystem-sdk/src/main/java/com/kin/ecosystem/history/presenter/OrderHistoryPresenter.java

```diff
@@ -59,49 +61,47 @@ public void onFailure(Throwable t) {
         });
     }
 
-    private void syncNewOrders(OrderList newOrdersList) {
-        if (orderHistoryList != null && orderHistoryList.getOrders() != null
-            && orderHistoryList.getOrders().size() > 0) {
-            List<Order> oldList = orderHistoryList.getOrders();
-            List<Order> newList = newOrdersList.getOrders();
+    private void syncNewOrders(OrderList newOrdersListObj) {
```

Comment:
you can pass List instead of OrderList.
If require you can add:
```
private List<Order> getOrderAsList(OrderList orderList){
if (orderList != null and orderList.getOrder() != null){
return orderList.getOrder()
} else {
return new ArrayList<Order>();
}
```
reuse it in `removePendingOrders()`